### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by IvanSkadorva**

### CODE REVIEW

File: src/app/footer/footer.component.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  &__list-header {
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 19px;
+    text-transform: uppercase;
+  }
+  &__list-of-options {
+    display: flex;
+    flex-direction: row;
+    margin-left: -40px;
```
Comment (marinbonibon) [inline src/app/footer/footer.component.scss:21]: It's bad to use negative margin

File: src/app/footer/footer.component.scss [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    justify-content: center;
+  }
+}
+
+@media screen and (max-width: 1200px) {
+  .footer {
+    height: 400px;
+
+    &__container {
+      justify-content: none;
```
Comment (marinbonibon) [inline src/app/footer/footer.component.scss:63]: It's invalid property value

File: src/app/footer/footer.component.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<footer class="footer">
+  <div class="footer__container">
+    <div class="footer__list-of-options-container ">
```
Comment (marinbonibon) [inline src/app/footer/footer.component.html:3]: It's better not to use preposition in names

File: src/app/footer/footer.component.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+          <li><a href="#">Terms of use</a></li>
+        </ul>
+        <ul>
+          <li><a href="#">About Us</a></li>
+          <li><a href="#">Q&A</a></li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="footer__app-download-icons-container">
```
Comment (marinbonibon) [inline src/app/footer/footer.component.html:25]: I think that it's not container for icons but for links to stores, think about some another name

File: src/app/footer/footer.component.html [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      <a href="#"><img src="assets/images/App-Store-Outline-badge.svg"></a>
```
[the remark this answers is not in the document]
Comment (IvanSkadorva) [reply src/app/footer/footer.component.html:26]: They are the same as in design. I exported all images from Figma

File: src/app/footer/footer.component.scss (hunk 1 above)
Comment (IvanSkadorva) [reply src/app/footer/footer.component.scss:21]: Okay, I'll fix it!

File: src/app/footer/footer.component.html (hunk 5 above)
[the remark this answers is not in the document]
Comment (marinbonibon) [reply src/app/footer/footer.component.html]: Hm, but when I run your branch in browser, I noticed that they look a little bit different (borders)

File: src/app/footer/footer.component.scss [hunk 6]
```diff
@@ -0,0 +1,72 @@
+.footer {
+  font-family: SF Pro Display, Helvetica, Arial, san-serif;
```
[the remark this answers is not in the document]
Comment (IvanSkadorva) [reply src/app/footer/footer.component.scss:2]: I'm waiting for the merge to dev by Artyom's PR with added fonts

**Pull request merged**